lib/jsdom/living/xhr/FormData-impl.js

```diff
@@ -170,9 +169,9 @@ function constructTheFormDataSet(form, submitter) {
 }
 
 function isBlob(obj) {
-  return obj !== null && typeof obj === "object" && blobSymbols.buffer in obj;
+  return obj instanceof Blob;
```

Comment:
That's not true; Object.assign doesn't copy symbols. You really have to work at it. Right now we're using symbols as the best balance between performance and unforgeability; in the future maybe we can use real private state.